nimgen.nim

```diff
@@ -1,5 +1,11 @@
 import nre, os, ospaths, osproc, parsecfg, pegs, ropes, sequtils, streams, strutils, tables
 
+const
+  C_COMPILER_ENV = "CC"
+  CPP_COMPILER_ENV = "CPP"
+  DEFAULT_C_COMPILER = "gcc"
+  DEFAULT_CPP_COMPILER = "g++"
+
```

Comment:
Can we use camel case to adhere to [nep1](https://nim-lang.org/docs/nep1.html#introduction-naming-conventions)?

cCompilerEnv, etc. Shouldn't hurt to hard-code too since it isn't expected to change.

Reply:
Sure!